setup.py

```diff
@@ -98,7 +98,6 @@ def get_setup_kwargs(raw=False):
                 'Topic :: Internet',
                 'Programming Language :: Python',
                 'Programming Language :: Python :: 3',
-                'Programming Language :: Python :: 3.6',
```

Comment:
do the changes include anything that doesn't work on v3.6? If not, then IMO we can even keep support for 3.6. on v20 we explicitly use 3.7+ code so there it makes more sense.

Reply:
yes, cryptography fails.

Reply:
fair enough